src/components/ComputeModal.js

```diff
@@ -1348,6 +1348,7 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
             existingRuntime.tool === 'RStudio' ? h(SaveFilesHelpRStudio) : h(SaveFilesHelp)
           ])],
           [willRequireDowntime(), () => h(Fragment, [
+            existingRuntime.tool !== tool ? p([`By continuing, you will be changing the application of your cloud environment from ${existingRuntime.tool} to ${tool}.`]) : undefined,
```

Comment:
Should we bold-face the tool names so they are easier to catch on the message below?

![image](https://user-images.githubusercontent.com/5438223/152593980-4fd50de7-2815-4165-a2df-f69ce2ff2768.png)